The find command used in the upstream Makefile copies identical files
over eachother without forcing the copy.  This patch adds the -f (force)
option to the find copy command.  This is not explicitly needed but does
seem to reduce a warning.

--- a/src/spidermonkey/Makefile
+++ b/src/spidermonkey/Makefile
@@ -30,12 +30,12 @@

 jslib: js-buildstamp
 	cd js/src
-	find . -name "libjs.a" -exec cp {} .. \;
+	find . -name "libjs.a" -exec cp -f {} .. \;

 js-buildstamp:
 	mkdir -p js/src/$(OBJDIR)
 	CFLAGS="$(SMCFLAGS)" "$(MAKE)" -C js/src -f Makefile.ref libjs.a
-	find js/src -name "jsautocfg.h" -exec cp {} js/src \;
+	find js/src -name "jsautocfg.h" -exec cp -f {} js/src \;
 	touch js-buildstamp

 clean:
